homeassistant/components/niko_home_control/light.py

```diff
@@ -86,7 +144,7 @@ async def async_update(self) -> None:
         await self._data.async_update()
         state = self._data.get_state(self._light.id)
         self._attr_is_on = state != 0
-        if brightness_supported(self.supported_color_modes):
+        if brightness_supported(self._attr_supported_color_modes):
```

Comment:
?

Reply:
Yes well, I'm a python/hass noob :p,  I guess `self._attr_support...` can be get using `self.support`?

Reply:
No more like, why did we touch this?